package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Could we update the `pyproject.toml` instead of adding a `package.json`?

Reply:
Done! Have added `version` into the `pyproject.toml` and deleted package.json